packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.utils.ts

```diff
@@ -49,6 +50,10 @@ export default ({ coreSagas, networks }: { coreSagas: any; networks: any }) => {
         updatedPayment = yield updatedPayment.memoType('text')
         // @ts-ignore
         updatedPayment = yield updatedPayment.setDestinationAccountExists(true)
+      } else if (hotwalletAddress && payment.coin === 'ETH') {
```

Comment:
Will this also be used for ERC20 coins?

Reply:
Yes, in the case of ERC20's this payment.coin value is also 'ETH'.